pkg/generate/app/templatelookup.go

```diff
@@ -21,34 +21,38 @@ type TemplateSearcher struct {
 	Client                    client.TemplatesNamespacer
 	TemplateConfigsNamespacer client.TemplateConfigsNamespacer
 	Namespaces                []string
+	StopOnMatch               bool
```

Comment:
StopOnMatch or StopOnExactMatch?

if it's the latter, rename it or comment it.